pkg/apis/configuration/validation/virtualserver.go

```diff
@@ -409,9 +410,13 @@ func (vsv *VirtualServerValidator) validateUpstreams(upstreams []v1.Upstream, fi
 		} else {
 			upstreamNames.Insert(u.Name)
 		}
+		if u.UseClusterIP && u.Subselector != nil {
```

Comment:
would it make sense to extend unit tests to cover this new condition ?

Reply:
I had a feeling I was missing something! 😅